tasks.py

```diff
@@ -0,0 +1,206 @@
+        Assumes that the environment is already configured using:
+            conda devenv
+            activate autodiff
+        """
+        build_dir, artifacts_dir = _get_and_prepare_build(
+            c,
+            clean=clean,
+            build_subdirectory=BUILD_DIR_DEFAULT / "msvc",
+        )
+        cmake_command = _get_cmake_command(build_dir=build_dir, cmake_generator="Visual Studio 15 2017",
+                                           cmake_arch="x64", config=config)
+        os.chdir(build_dir)
+        c.run(cmake_command)
+
+
+@task
+def compile(c, clean=False, config='Release', number_of_jobs=-1, verbose=False, gen_wrappers=False):
+    """
+    Compiles autodiff by running CMake and building with `ninja`.
+    Assumes that the environment is already configured using:
+        conda devenv
+        [source] activate autodiff
+    """
+    build_dir = _get_and_prepare_build(
+        c,
+        clean=clean,
+        build_subdirectory=BUILD_DIR_DEFAULT,
+    )
+
+    cmake_command = _get_cmake_command(build_dir=build_dir, cmake_generator="Ninja", config=config, verbose=verbose)
+    build_command = strip_and_join(f"""
+        cmake
+            --build .
+            --target install
+            --config {config}
+            --
+                {f"-j {number_of_jobs}" if number_of_jobs >= 0 else ""}
+                {"-d keeprsp" if sys.platform.startswith("win") else ""}
+    """)
+
+    commands = [cmake_command, build_command]
+    if gen_wrappers:
+        wrappers_command = _get_wrappers_command(build_dir / "wrappers/conda")
+        commands.append(wrappers_command)
+
+    if sys.platform.startswith('win'):
+        for vcvars_path in _get_vcvars_paths():
+            if not vcvars_path.is_file():
+                continue
+            commands.insert(0, f'"{vcvars_path}" amd64')
+            break
+        else:
+            raise Exit(
+                'Error: Commands to configure MSVC environment variables not found.',
+                code=1,
+            )
+
+    os.chdir(build_dir)
+    c.run("&&".join(commands))
+
+
+@task
+def clear(c, build_dir_path=BUILD_DIR_DEFAULT):
+    """
+    Clear autodiff build directory
+    """
+    remove_directory(build_dir_path)
+
+
+@task
+def wrappers(c, wrappers_dir=BUILD_DIR_DEFAULT / "wrappers/conda"):
+    """
+    Wrappers bin generated by autodiff conda environment as passed with --wrappers-dir dir_path
+    """
+    remove_directory(wrappers_dir)
+    if sys.platform.startswith('win'):
+        print(f"Generating conda wrappers to {wrappers_dir} from {os.environ['CONDA_PREFIX']}\\Library\\bin")
+    else:
+        print(f"Generating conda wrappers to {wrappers_dir} from {os.environ['CONDA_PREFIX']}/bin")
+
+    generate_wrappers_command = _get_wrappers_command(wrappers_dir)
+    echo(c, generate_wrappers_command)
+    c.run(generate_wrappers_command, pty=True, warn=True)
+
+
+@task
+def tests(c):
```

Comment:
The `tests` task runs `autodiff`'s `Catch2` tests with a "pretty" output, close to `pytest`. It is simply the following command behind the scenes: `{test_executable} -s -r compact`. Please note that there is no need to know where is the `{test_executable}`, it is encapsulated and configure inside the task.

Usage:
```
inv tests
```

No options are available at this point.